vrecord_policy.xml

```diff
@@ -0,0 +1,93 @@
+            </policy>
+            <policy type="or" name="Is it NTSC SD or PAL SD?">
+                <policy type="and" name="Is it NTSC?">
+                    <rule name="NTSC FrameRate 29.970?" value="FrameRate" tracktype="Video" operator="=">29.970</rule>
+                </policy>
+                <policy type="and" name="Is it PAL?">
+                    <description>The PAL matrix coefficients value is reported differently depending on container. This test validates for Quicktime</description>
+                    <rule name="PAL FrameRate 25.000?" value="FrameRate" tracktype="Video" operator="=">25.000</rule>
+                    <rule name="Are the matrix coefficients BT.601?" value="matrix_coefficients" tracktype="Video" operator="=">BT.601</rule>
+                </policy>
+            </policy>
+            <rule name="Is the video format FFV1?" value="Format" tracktype="Video" operator="=">FFV1</rule>
+            <rule name="FFV1 is version 3.4 or later?" value="Format_Version" tracktype="Video" operator="&gt;=">3.4</rule>
+            <rule name="FFV1 is encoded in GOP size of 1?" value="Format_Settings_GOP" tracktype="Video" operator="=">N=1</rule>
+            <rule name="Is the display aspect ratio 1.333?" value="DisplayAspectRatio" tracktype="Video" operator="=">1.333</rule>
+            <rule name="Is the frame rate mode constant?" value="FrameRate_Mode" tracktype="Video" operator="=">CFR</rule>
+            <rule name="Is the color space YUV?" value="ColorSpace" tracktype="Video" operator="=">YUV</rule>
+            <rule name="Is the chroma subsampling 4:2:2?" value="ChromaSubsampling" tracktype="Video" operator="=">4:2:2</rule>
+            <rule name="Is the video interlaced?" value="ScanType" tracktype="Video" operator="=">Interlaced</rule>
+            <rule name="Is the comression mode lossless?" value="Compression_Mode" tracktype="Video" operator="=">Lossless</rule>
+            <rule name="Is max slice count at least 4?" value="extra/MaxSlicesCount" tracktype="Video" operator="&gt;=">4</rule>
+            <rule name="Is error detection type per slice?" value="extra/ErrorDetectionType" tracktype="Video" operator="=">Per slice</rule>
+            <rule name="Is the video stream first?" value="StreamOrder" tracktype="Video" operator="=">0</rule>
+            <rule name="Is the colour description present?" value="colour_description_present" tracktype="Video" operator="=">Yes</rule>
+            <rule name="Are the transfer characteristics BT.709?" value="transfer_characteristics" tracktype="Video" operator="=">BT.709</rule>
+            <policy type="or" name="Is the bit depth 8 or 10?">
+                <rule name="Is the bit depth 8?" value="BitDepth" tracktype="Video" operator="=">8</rule>
+                <rule name="Is the bit depth 10?" value="BitDepth" tracktype="Video" operator="=">10</rule>
+            </policy>
+            <policy type="or" name="Is it NTSC SD or PAL SD?">
```

Comment:
I wonder if the previous NTSC or PAL checks could all be moved in here? Looks like there is some duplication.

Reply:
I wasn't quite sure  the best way to go about this.... because the matrix coefficients values are different for PAL depending on whether the format is quicktime or matroska, there needs to be redundant testing somewhere in the policy. So the reason the same test exists at line 14 as 34 is because one is for matroska and the other quicktime. 

I'm happy to move that testing down into the section starting at line 62, but then I would need to add in duplicate tests there for matroska and quicktime to deal with the differing PAL matrix coefficients. But maybe that's more efficient? 

I'm new to this!

Reply:
Ah, this makes sense now - I remember seeing this in your issue https://github.com/amiaopensource/vrecord/issues/193 - I can see why you had to work around it like this.. I have nothing else to add as I'm quite unfamiliar with the and/or policies so I'm not sure how best to proceed, i think what you did looks good to me , though.